keyboards/bemeier/bmek/rev3/rules.mk

```diff
@@ -0,0 +1 @@
+LAYOUTS = all_67 hhkb
```

Comment:
This file should be present, but empty.

```suggestion
```

Reply:
Oh I thought this also mattered for the configurator. Either way, happy to remove it :)